src/modules/robot/arm_solutions/MorganSCARASolution.cpp

```diff
@@ -58,7 +60,7 @@ float MorganSCARASolution::to_degrees(float radians) const
 
 void MorganSCARASolution::cartesian_to_actuator(const float cartesian_mm[], ActuatorCoordinates &actuator_mm ) const
 {
-
+  if (!THEROBOT->disable_arm_solution){
```

Comment:
I have proven it to be otherwise.  I put a stream print function in the
arm_solution cartesian_to_actuator, and actuator_to_cartesian and during
homing these commands gets called a number of times.


On Mon, Aug 29, 2016 at 8:40 PM Jim Morris <notifications@github.com> wrote:

> In src/modules/robot/arm_solutions/MorganSCARASolution.cpp
> <https://github.com/Smoothieware/Smoothieware/pull/1005#discussion_r76662334>
> :
>
> > @@ -58,7 +60,7 @@ float MorganSCARASolution::to_degrees(float radians) const
> >
> >  void MorganSCARASolution::cartesian_to_actuator(const float cartesian_mm[], ActuatorCoordinates &actuator_mm ) const
> >  {
> > -
> > +  if (!THEROBOT->disable_arm_solution){
>
> This is wrong. there are no changes to the arm solution. when
> disable-arm-solution is set the arm solution is never called by robot.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Smoothieware/Smoothieware/pull/1005/files/fc61058c44f0d4719f5c74cae295bd0d07d787cf#r76662334>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0PWNbvjhujD_rKHlM988NQBx9QarQxks5qkydggaJpZM4Jph12>
> .
>